### PR TITLE
Login

### PR DESCRIPTION
**Pull request by yael-renous**
login to facebook and retrieves posts from group

### CODE REVIEW

**Review by omerjacobi — CHANGES_REQUESTED** [review]
we need to fill the code of LoginResult
onCancel
onError

File: app/src/main/java/com/huji/cse/flatfinder/MainActivity.java [hunk 1]
```diff
@@ -2,12 +2,73 @@
...
+        LoginManager.getInstance().registerCallback(callbackManager,
+                new FacebookCallback<LoginResult>() {
+                    @Override
+                    public void onSuccess(LoginResult loginResult) {
+                        // App code
+                        get_posts_in_graph(loginResult);
+                    }
+
+                    @Override
+                    public void onCancel() {
```
Comment (m-mandel) [inline app/src/main/java/com/huji/cse/flatfinder/MainActivity.java:42]: is it really necessary?

File: app/src/main/java/com/huji/cse/flatfinder/MainActivity.java [hunk 2]
```diff
@@ -2,12 +2,73 @@
...
+                    }
+
+                    @Override
+                    public void onError(FacebookException exception) {
+                        // App code
+                    }
+                });
+    }
+
+    private void get_posts_in_graph(LoginResult loginResult) {
```
Comment (m-mandel) [inline app/src/main/java/com/huji/cse/flatfinder/MainActivity.java]: Conventions!!! Use camel case..

File: app/src/main/java/com/huji/cse/flatfinder/MainActivity.java [hunk 3]
```diff
@@ -2,12 +2,73 @@
...
+                token,
+                new GraphRequest.GraphJSONObjectCallback() {
+                    @Override
+                    public void onCompleted(
+                            JSONObject object,
+                            GraphResponse response) {
+                        grape_posts_object = object;
+                    }
+                });
+        request.setGraphPath("337906013661346/feed?fields=link,message,created_time,name,full_picture,id");
```
Comment (m-mandel) [inline app/src/main/java/com/huji/cse/flatfinder/MainActivity.java]: why not constant?

File: .idea/gradle.xml [hunk 4]
```diff
@@ -3,9 +3,8 @@
   <component name="GradleSettings">
```
Comment (m-mandel) [inline .idea/gradle.xml]: 👍

**Review by m-mandel — CHANGES_REQUESTED** [review]
Hi ~ good job~ 🥇  fix and send it again...

File: app/src/main/java/com/huji/cse/flatfinder/MainActivity.java (hunk 1 above)
Comment (m-mandel) [reply app/src/main/java/com/huji/cse/flatfinder/MainActivity.java:42]: yes.. it won;t work

**Review by omerjacobi — COMMENTED** [review]
greate job

**Comment by omerjacobi** [comment]
finish code review on facebook login

**Pull request merged**